app/konnect-reference-platform/faq.md

```diff
@@ -0,0 +1,224 @@
+
+## What are the main Konnect Orchestrator concepts?
+
+<u>Declarative Config</u>:
+The orchestrator accepts YAML files that make up its declarative configuration, which it reads and reconciles to Konnect organizations.
+
+<u>Platform Team</u>:
+The Platform Team is a special team within KO that represents a typical Platform Team that own the API Platform / API Delivery system 
+within a broader engineering organization. KO accepts a distinct Platform Team Git repository configuration where much of the configuration is 
+written to and the APIOps processes work from.
+
+<u>Organizations</u>:
+KO can manage 1-n organizations which define the intersection of the Developer Teams and their Application Services, to 
+Konnect Organizations and the resources defined inside them.
+
+<u>Teams</u>:
+Developer Teams / Application Teams
+
+<u>Service Applications</u>:
+These are the Service Applications owned by the Developer Teams. Otherwise known as “Backend Services” or “Upstream Applications”
+
+## Where can I find an example of the Reference Platform in action?
+The example organization, _Kong Airlines_, provides a fictional airline engineering solution made up of a 
+set of teams and service applications that support common airline concepts (flights, bookings, etc…) and a Platform
+team that manages the API delivery process. The example provides a sample declarative configuration and utilizes the
+API Delivery process in the platform repository. 
+
+[Kong Airlines is available on GitHub](https://github.com/KongAirlines).
+
+## How does the Konnect Orchestrator work?
+The orchestrator follows the below general pattern:
+
+* Read the input declarative configuration from file(s)
+* Manage the APIOps Workflows within the Platform repository by staging Pull Requests that keep API delivery workflow automation up to date
+* Directly adds or modifies Konnect resources in one or more Konnect organizations based on the input configuration
+* Read API Specifications (OpenAPI files) from Service Application repositories and stage them into the Platform repository via Pull Requests
+
+## What Konnect resources are managed by the Konnect Orchestrator?
+* Teams, Roles, and Users including invitations and user -> team mappings
+* IdP configuration including Konnect Built-in, OIDC, SAML, and IdP Group to Konnect Team mappings
+* Control Planes and Control Plane RBAC policies for managed Teams
+* Developer Portals and APIs including API specifications and API Implementations
+* Analytics Custom Reports
+* Notification Hub configurations 
+
+## What is the specific role of the Platform Team in the Reference Platform?
+The Reference Platform operates off of a central code repository, the _Platform Team_ repository. This is by design, such that organizations
+can on-board application teams without having to "inject" and code or process into the application team code, repositories or engineering workflows.
```

Comment:
```suggestion
The Reference Platform operates off of a central code repository, the _Platform Team_ repository. This is designed so that organizations
can onboard application teams without having to "inject" and code or process into the application team code, repositories, or engineering workflows.
```